public/scripts/app.js

```diff
@@ -0,0 +1,151 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var calc = new Calculator();
+  var calcButtons = document.getElementsByClassName('calc-display-num')[0];
+  calcButtons.addEventListener('click', function(event) {
+    calc.buttonPressed(event);
+  })
+})
+
+function Calculator() {
+  this.firstNums = '';
+  this.decimalCheck = false;
+  this.total = null;
+  this.operator = null;
+  this.operand = null;
+  this.saveOperation = false;
+  this.overwrite = false;
+}
+
+Calculator.prototype.displayNums = function(innerText) {
+  this.firstNums += innerText;
+  this.updateDisplay();
+}
+
+// calculator match function
+Calculator.prototype.buttonPressed = function(event) {
+  switch(event.target.innerText) {
```

Comment:
The good thing about using `innerText` here is that the cases read exactly like the buttons themselves.

Some of them, particularly `+/-` might be difficult to translate to from a keyDown event.